src/tests/test_api.py

```diff
@@ -91,6 +85,71 @@ def test_invalid_method(self):
             'API Error 0020: Invalid method: test'
         )
 
+    def test_statusbar_method(self):
+        """Test statusbar method"""
+        self.api.clearUISignalQueue()
+        self.assertEqual(
+            self.api.statusBar(sample_statusbar_msg),
+            'null'
+        )
+        self.assertEqual(
+            self.api.getStatusBar(),
+            sample_statusbar_msg
+        )
+
+    def test_message_inbox(self):
+        """Test message inbox methods"""
+        self.assertEqual(
+            len(json.loads(
+                self.api.getAllInboxMessages())["inboxMessages"]),
+            4,
+            json.loads(self.api.getAllInboxMessages())["inboxMessages"]
```

Comment:
Maybe add a comment explaining this is the custom error message to aid in debugging.